### PR TITLE
Implement GitHub Actions for Testing & Deployment

### PR DESCRIPTION
**Pull request by CalumTowers**
This PR;

- Adds a test that passes, just so PHPUnit passes (no tests will result in incomplete/failure)
- Adds a GitHub action that sets up the world, installs dependencies, serves the app and runs the test suite
- Adds a GitHub action to automatically deploy the app to a remote server on `main`

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        uses: actions/cache@v1
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      # Install
+      - name: Install Core Assets
```
Comment (AxonC) [inline .github/workflows/test.yml]: Is this core as in important or core as in the name? If so, might need renaming

File: tests/Feature/PassTest.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Tests\TestCase;
+
+class PassTest extends TestCase
+{
+    /** @test */
+    public function it_passes_phpunit()
+    {
+        $this->assertTrue(true);
```
Comment (AxonC) [inline tests/Feature/PassTest.php:10]: Still more tests than some apps 💯

**Review by AxonC — DISMISSED** [review]

File: tests/Feature/PassTest.php (hunk 2 above)
Comment (CalumTowers) [reply tests/Feature/PassTest.php:10]: Most reliable test I've ever written!

File: .github/workflows/test.yml (hunk 1 above)
Comment (CalumTowers) [reply .github/workflows/test.yml]: Was a typo! Now corrected

**Review by AxonC — APPROVED** [review]

**Pull request merged**